### DISCUSSION TITLE
Enter press when MentionsPopover is open

### DISCUSSION THREAD

**Discussion by ankitsharma-99** [facebook/lexical#4280] [Q&A] [answered]
I am trying to use MentionsPlugin in the Editor. I want to be able to submit a comment on enter press, but when the mentions popover is open, the enter key press should just select the highlighted item and close the mentions popover , i.e, the comment shouldn't be submitted when mentions popover is open . How do I achieve this?

**Comment by moy2010** [comment, marked as answer]
Use the `stopPropagation` method on the keyboard event.

**Reply by ankitsharma-99** [reply]
Thanks! That works.
